src/GraphQL/Validation/ValidationError.cs

```diff
@@ -42,7 +42,7 @@ public ValidationError(string originalQuery, string number, string message, para
             nodes?.Apply(n =>
             {
                 _nodes.Add(n);
-                if (n.SourceLocation != null)
+                if (n.SourceLocation != SourceLocation.Empty)
                 {
                     var location = new Location(new Source(originalQuery), n.SourceLocation.Start);
```

Comment:
This line of code calls `GraphQLParser` to retrieve the `Line` and `Column` from the query based on the start position within the document.